UtilitsforBioinfApplication/helpertools/dna_rna_tools_module.py

```diff
@@ -0,0 +1,61 @@
+def transcr(seq):  # accepts a string - a sequence of nucleotides of any length
+    newseq = ''
+    for nucl in seq:
+        if nucl == 'T':
+            newseq += 'U'
+        elif nucl == 't':
+            newseq += 'u'
+        else:
+            newseq += nucl
+    return (newseq)  # returns a string - a sequence translated to RNA form
+
+
+def reverse(seq):  # accepts a string - a sequence of nucleotides of any length
+    newseq = ''
+    newseq = seq[::-1]
+    return (newseq)  # returns a string - a reversed sequence
+
+
+def compl(seq):   # accepts a string - a sequence of nucleotides of any length
+    newseq = ''
+    dna_comp_dict = {'A': 'T', 'a': 't', 'T': 'A', 't': 'a', 'C': 'G', 'c': 'g', 'G': 'C', 'g': 'c'}
+    rna_comp_dict = {'A': 'U', 'a': 'u', 'U': 'A', 'u': 'a', 'C': 'G', 'c': 'g', 'G': 'C', 'g': 'c'}
+    res_test = NA_type(seq)
+    if res_test == 'DNA':
+        newseq = ''.join([dna_comp_dict[nucl] for nucl in seq])
+    else:
+        newseq = ''.join([rna_comp_dict[nucl] for nucl in seq])
+    return (newseq)  # reterns a string - a comlement of given sequence
+
+
+def rev_compl(seq):  # accepts a string - a sequence of nucleotides of any length
+    res1 = reverse(seq)
+    res = compl(res1)
+    return (res)  # returns a string - a complement of reversed given sequence
+
+
+def NA_type(seq):  # accepts a string - a sequence of nucleotides of any length
```

Comment:
In the name of function we expect to see the verb 
```suggestion
def get_NA_type(seq):  # accepts a string - a sequence of nucleotides of any length
```

Reply:
Респект за поддержание англоязычной атмосферы в Pull-request'е! Здесь, понятно, учебная история, но в целом в каком-нибудь реальном PR скорее всего будут люди из разных стран.